react-task/src/App.js

```diff
@@ -1,25 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Register from "./components/Register";
```

Comment:
Use this structure for import ordering

```suggestion
import React from 'react';

import Register from "./components/Register";

import "./App.css";
```